app/src/main/admin/components/AdminRouter.tsx

```diff
@@ -27,7 +26,7 @@ interface AdminRouterProps {
 export const AdminRouter : React.StatelessComponent<AdminRouterProps> = (props: AdminRouterProps) => {
 
     const loggedIn = <Switch>
-        <Route exact path="/" component={MainMenu}/>
+        <Route exact path="/" component={MainMenuNew}/>
```

Comment:
We should also go through and rename all the `xNew` components to `x`, but maybe there's another PR for this?

Reply:
Yup, that's the last PR